ecs/client.go

```diff
@@ -103,6 +102,32 @@ func (c *BasicECSClient) RegisterTaskDefinition(ctx context.Context, in *ecs.Reg
 	return out, nil
 }
 
+// ListTaskDefinitions returns the ARNs for the task definitions that match the
+// input filters.
+func (c *BasicECSClient) ListTaskDefinitions(ctx context.Context, in *ecs.ListTaskDefinitionsInput) (*ecs.ListTaskDefinitionsOutput, error) {
+	if err := c.setup(); err != nil {
+		return nil, errors.Wrap(err, "setting up client")
+	}
+
+	var out *ecs.ListTaskDefinitionsOutput
```

Comment:
We don't do this anywhere else right now. I think it's okay to leave as it is since it's only 2 variables. If we need to declare more, we can use `var ()`.